.github/workflows/tests.yml

```diff
@@ -0,0 +1,65 @@
+name: Test Suite
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+
+jobs:
+    test:
+        runs-on: ${{matrix.operating-system}}
+        strategy:
+            matrix:
+                operating-system: [ ubuntu-latest ]
+                php: [ "5.6", "7.0", "7.1", "7.2", "7.3", "7.4" ]
```

Comment:
Should we add PHP 8.0?

Reply:
I'd be ok holding off until it's released

Reply:
(mostly because it's failing right now)

Reply:
fine with me then.